test-end-to-end/android-maestro/e2e-tests/002_009_people_add_and_remove_profile_pic.yaml

```diff
@@ -0,0 +1,24 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTNAME: "002_009_people_add_and_remove_profile_pic"
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/002_009_people_add_and_remove_profile_pic"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+
+- runFlow:
+    file: "../common/add_profile_pic.yaml"
+    env:
+      TESTFILENAME: "admin_pic.jpg"
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+- tapOn:
```

Comment:
This needs an assertion that the picture is visible after being set e.g. open the profile and then make an assertion